File: URI_1019.py
N = int(input())
horas = N / 3600
minutos, segundos = 0

if horas < 0:
    horas = 0
else:
    minutos = N % 3600

minutos = minutos / 60
if minutos < 0:
    minutos = 0
else:
    segundos = N % 60

print("%d:%d:%d" % (int(horas), int(minutos), segundos))

File: URI_1019.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

    Scanner input = new Scanner(System.in);

    int N, horas, minutos;

    N = input.nextInt();

    horas = N / 3600;
    N -= 3600 * horas;
    minutos = N / 60;
    N -= 60 * minutos;


    System.out.println(horas + ":" + minutos + ":" + N);

    }
}
